ang/test/civicase/CaseDetails.spec.js

```diff
@@ -64,13 +64,14 @@ describe('civicaseCaseDetails', function () {
       compileDirective();
       element.isolateScope().item = CasesData.values[0];
       element.isolateScope().item.allActivities = CasesData.values[0]['api.Activity.get.1'].values;
-
+      element.isolateScope().item.nextActivities = CasesData.values[0]['api.Activity.get.4'].values;
       element.isolateScope().pushCaseData(CasesData.values[0]);
     });
 
-    it('calculates the incomplete tasks and scheduled activities', function () {
+    it('calculates the incomplete tasks and scheduled activities and nextActivitiyWhichIsNotMilestone', function () {
```

Comment:
Please have 3 separate `it` for each of the `expect` statements.

Reply:
Since the next activity is coming from the API response we don't need to write tests for it.